src/lib/rest/rest.cpp

```diff
@@ -1168,6 +1182,8 @@ static int restStart(IpVersion ipVersion, const char* httpsKey = NULL, const cha
                                    MHD_OPTION_HTTPS_MEM_KEY,            httpsKey,
                                    MHD_OPTION_HTTPS_MEM_CERT,           httpsCertificate,
                                    MHD_OPTION_CONNECTION_MEMORY_LIMIT,  memoryLimit,
+                                   MHD_OPTION_CONNECTION_LIMIT,         maxConns,
+                                   MHD_OPTION_THREAD_POOL_SIZE,         threadPoolSize,
```

Comment:
What it should be great is to have a set of env vars to model different "working models", e.g. w/ or w/o cache, w/ or w/o threadd pool, with the different notification modes, etc. and configure jenkins to do a pass in all them.

Issue about it is on the way...